### ISSUE TITLE
Feature request: tournament seed change notification

### ISSUE THREAD

**Issue by RichieRoxx** [raphi011/scores-api#34]
from the moment you signed up for a tournament on, any change in team seed is sent by email to the team which has signed up via scores.

**Comment by raphi011** [comment]
good idea! there should be atleast some optin/out option or delayed emails, so the users inbox doesn't get flooded
